### PR TITLE
doc: Update AWS policies README.md

### PR DESCRIPTION
**Pull request by hermanschaaf**

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]
Looks great. There's another way to get the policies by downloading the plugin zip (any of them) and it should have a `policies` folder with the relevant policies

File: plugins/source/aws/policies/README.md [hunk 1]
```diff
@@ -10,6 +10,27 @@ CloudQuery SQL Policies for AWS
...
 
+## Installing
+
+Clone this repository locally:
+
+```bash
+git clone https://github.com/cloudquery/cloudquery.git cloudquery
+```
+
+Check out the tag matching the AWS Source plugin version you are using, so that the policy queries match the table schemas:
```
Comment (hermanschaaf) [inline plugins/source/aws/policies/README.md:21]:
```suggestion
Check out the tag matching the AWS Source plugin version you are using:
```

**Review by disq — APPROVED** [review]

**Comment by hermanschaaf** [comment]
Thanks @erezrokah, good point. I added a section about how to do it via Releases as well, if you'd like to take another look.

File: plugins/source/aws/policies/README.md [hunk 2]
```diff
@@ -10,6 +10,39 @@ CloudQuery SQL Policies for AWS
...
+### Option 2: Downloading via Releases
+
+This is an alternative method to get the correct policies for a given AWS source plugin version, and is a better fit for automation.
+
+1. Go to https://github.com/cloudquery/cloudquery/releases?q=plugins-source-aws&expanded=true
+2. Find the release version that matches the AWS version in your source config
+3. Click on "Assets" to expand the list
+4. Download the Source Code zip file and unzip it.
+5. Change directory into policies: `cd plugins/source/aws/policies`
+
```
Comment (erezrokah) [inline plugins/source/aws/policies/README.md:45]: Maybe we can add `curl -iL https://github.com/cloudquery/cloudquery/releases/download/plugins-source-aws-v16.3.0/aws_linux_amd64.zip -o aws.zip && unzip aws.zip "policies/*"` as a single line command to do this?

**Review by erezrokah — APPROVED** [review]
Looks good. Added one non blocking comment

File: plugins/source/aws/policies/README.md [hunk 3]
```diff
@@ -10,6 +10,39 @@ CloudQuery SQL Policies for AWS
...
+### Option 2: Downloading via Releases
+
+This is an alternative method to get the correct policies for a given AWS source plugin version, and is a better fit for automation.
+
+1. Go to https://github.com/cloudquery/cloudquery/releases?q=plugins-source-aws&expanded=true
+2. Find the release version that matches the AWS version in your source config
+3. Click on "Assets" to expand the list
+4. Download the Source Code zip file and unzip it.
+5. Change directory into policies: `cd plugins/source/aws/policies`
```
Comment (erezrokah) [inline plugins/source/aws/policies/README.md:44]:
```suggestion
5. Change directory into policies: `cd policies`
```

File: plugins/source/aws/policies/README.md (hunk 2 above)
Comment (hermanschaaf) [reply plugins/source/aws/policies/README.md:45]: Ah nice, yeah let's do that

File: plugins/source/aws/policies/README.md (hunk 3 above)
Comment (erezrokah) [reply plugins/source/aws/policies/README.md:44]: The zip has the policies directory as a top level one

**Review by erezrokah — APPROVED** [review]

**Pull request merged**